runtime/src/accounts.rs

```diff
@@ -649,6 +646,16 @@ impl AccountsDB {
         parents
     }
 
+    fn is_squashed(&self, fork: Fork) -> bool {
+        self.fork_info
```

Comment:
This is kind of more expensive than the old way, but maybe it doesn't matter.

Reply:
what's the old way?

Reply:
checking the parent pointer on the bank.

Reply:
the duplication of state/logic was bugging me ;)